adafruit_circuitplayground/express.py

```diff
@@ -184,6 +191,30 @@ def red_led(self):
     def red_led(self, value):
         self._led.value = value
 
+    def play_tone(self, frequency, duration):
+        """ Produce a tone using the speaker.
+
+        .. image :: /_static/speaker.jpg
```

Comment:
Empty lines are weirdly important so you'll need one after the image line and after the paragraph below.